code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -40,7 +40,7 @@
 		return
 	has_suit.overlays -= inv_overlay
 	has_suit = null
-	src.loc = get_turf(user || src)
+	src.forceMove(get_turf(user || src))
```

Comment:
How so?
If you're referring to the use of `||`, remember that this is a weakly-typed language
`a || b` is a shortcut for `a ? a : b`